acme/challenge.go

```diff
@@ -173,6 +197,25 @@ func (bc *baseChallenge) getError() *AError {
 	return bc.Error
 }
 
+// getBackoff returns the backoff time of the baseChallenge.
+func (bc *baseChallenge) getBackoff() time.Duration {
+	if bc.Retry.Called == 0 {
+		return bc.Retry.BaseDelay
+	}
+
+	backoff, max := float64(bc.Retry.BaseDelay), float64(bc.Retry.MaxDelay)
+	backoff *= math.Pow(bc.Retry.Multiplier, bc.Retry.Called)
+	if backoff > max {
+		backoff = max
+	}
+	// Introduce Jitter to ensure that clustered requests wont operate in unison
```

Comment:
Not sure this matters either because we're not doing anything on a periodic basis. Only scenario where this might apply is if some client does a large batch of challenges all at once. In this case I think we would delegate managing that scenario to rate-limiting code.